diff --git a/cjs/client-side-base-visitor.js b/cjs/client-side-base-visitor.js
index a25a3f606da2110faba0dbc411b1e2079ff7050d..dd593f52e0988632a209b13c118f43dc3ed55354 100644
--- a/cjs/client-side-base-visitor.js
+++ b/cjs/client-side-base-visitor.js
@@ -164,7 +164,7 @@ class ClientSideBaseVisitor extends base_visitor_js_1.BaseVisitor {
             return `{${metaString}"kind":"${graphql_1.Kind.DOCUMENT}","definitions":${jsonStringify(definitions)}}`;
         }
         if (this.config.documentMode === DocumentMode.string) {
-            return '`' + doc + '`';
+            return '`' + graphql_1.stripIgnoredCharacters(doc) + '`';
         }
         const gqlImport = this._parseImport(this.config.gqlImport || 'graphql-tag');
         return (gqlImport.propName || 'gql') + '`' + doc + '`';