diff --git a/cjs/client-side-base-visitor.js b/cjs/client-side-base-visitor.js
index 9bd92e58298fd01a509e84273e34923d536d121f..eeb63ff3443fdc6ce749ad9eec0db03a1ca6e3cc 100644
--- a/cjs/client-side-base-visitor.js
+++ b/cjs/client-side-base-visitor.js
@@ -137,7 +137,7 @@ class ClientSideBaseVisitor extends base_visitor_js_1.BaseVisitor {
             return JSON.stringify(gqlObj);
         }
         if (this.config.documentMode === DocumentMode.string) {
-            return '`' + doc + '`';
+            return '`' + graphql_1.stripIgnoredCharacters(doc) + '`';
         }
         const gqlImport = this._parseImport(this.config.gqlImport || 'graphql-tag');
         return (gqlImport.propName || 'gql') + '`' + doc + '`';